src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/RequestExecutionQueue.cs

```diff
@@ -89,6 +90,19 @@ public void Start()
         return handler;
     }
 
+    /// <summary>
+    /// Indicates this queue requires in-progress work to be cancelled before servicing
+    /// a mutating request.
+    /// </summary>
+    /// <remarks>
+    /// This was added for WebTools consumption as they aren't resilient to
+    /// incomplete requests continuing execution during didChange notifications. As their
+    /// parse trees are mutable, a didChange notification requires all previous requests
+    /// to be completed before processing. This is similar to the O#
+    /// WithContentModifiedSupport(false) behavior.
```

Comment:
Is this something that WebTools would be fixing or changing at some point? The phrasing here makes this sound like this is somewhat workaround for the current state, but at some point would we no longer need to support this?

Reply:
No, this wording was added here because early in the PR Cyrus was curious about why this was needed and wanted it added to a comment somewhere so no-one would get rid of this code even if no-one in Razor or Roslyn use this feature.

Reply:
Resolving, let me know if you would like the comment changed.